c2corg_api/models/document_topic.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import (
+    Column,
+    ForeignKey,
+    Integer,
+    )
+from sqlalchemy.orm import relationship, backref
+
+from c2corg_api.models import schema, Base
+from c2corg_api.models.document import DocumentLocale
+
+
+class DocumentTopic(Base):
```

Comment:
Could you add a basic description what this class/table is for? E.g "Mapping between document locales and the corresponding Discourse topics for the comments."